scipy/stats/_common.py

```diff
@@ -1,6 +1,140 @@
 
 from collections import namedtuple
-
+from numpy import arange, newaxis, hstack, prod, array
 
 ConfidenceInterval = namedtuple("ConfidenceInterval", ["low", "high"])
 ConfidenceInterval. __doc__ = "Class for confidence intervals."
+
+
+def _central_diff_weights(Np, ndiv=1):
```

Comment:
It looks like it would be better to move this to `_lib` (e.g., `scipy/_lib/_finite_differences.py`), and then use it from both `stats` and `misc` - that way the code itself isn't duplicated. Also, `_lib` is a good place for this code to live, since it's not specific to `stats`. We have finite differences code in `optimize` too that may need dealing with at some point, and that'd be easier if it's in `_lib`.

Reply:
Made the move to `scipy/_lib/_finite_differences.py` and now using that everywhere else to avoid duplication.